sockeye/model.py

```diff
@@ -317,6 +317,17 @@ def load_parameters(self,
                                 cast_dtype=cast_dtype, dtype_source=dtype_source)
         logger.info('Loaded params from "%s" to "%s"', filename, mx.cpu() if ctx is None else ctx)
 
+    def set_parameters(self, new_params: dict):
```

Comment:
This method is very similar to `load_parameters()` from `mx.gluon.Block`, except that it does not load from disk.
Would it make sense to support some of its flags as well (i.e. `allow_missing`, `ignore_extra`)?
```python
def load_parameters(self, filename, ctx=None, allow_missing=False, ignore_extra=False,
    cast_dtype=False, dtype_source='current')```

Reply:
Good idea -- this now has `allow_missing` and `ignore_extra` flags.